src/components/LegalBanner.tsx

```diff
@@ -72,17 +82,24 @@ const LegalText: React.FC = () => (
   </Text>
 )
 
-const LegalBanner: React.FC<LegalBannerProps> = ({ startOpen, useFull, title }) => {
-  const [open, setOpen] = useState(startOpen)
+const DISCLAIMER = 'GP_DISCLAIMER'
+
+const LegalBanner: React.FC<LegalBannerProps> = ({ useFull, title }) => {
+  const [open, setOpen] = useState(JSON.parse(localStorage.getItem(DISCLAIMER) as string))
```

Comment:
It starts closed, as `localStorage.getItem(DISCLAIMER) === null` when nothing is yet in localStorage

Reply:
true
good point
oops

Reply:
i'll rock with strings